rotary/rules/rotary.smk

```diff
@@ -19,13 +19,25 @@ START_HMM_NAME = os.path.splitext(os.path.basename(config.get("hmm_url")))[0]
 VERSION_GTDB_COMPLETE= "214.1" # See https://data.gtdb.ecogenomic.org/releases/
 VERSION_GTDB_MAIN=VERSION_GTDB_COMPLETE.split('.')[0] # Remove subversion
 DB_DIR_PATH = config.get('db_dir')
-CONTAMINANT_REFERENCE_GENOMES_NCBI=get_contamination_reference_file_paths(
-    config.get("contamination_references_ncbi_accessions"), DB_DIR_PATH)
+
 SAMPLE_TSV_PATH = 'samples.tsv'
 SAMPLES = parse_sample_tsv(SAMPLE_TSV_PATH)
 
 SAMPLE_NAMES = list(SAMPLES.keys())
 
+CONTAMINATION_NCBI_ACCESSIONS = config.get("contamination_references_ncbi_accessions")
```

Comment:
@jmtsuji Note that once we start breaking the snakemake file into multi files we can move this logic to the top of the QC module.

Reply:
Agreed -- this sounds like a good idea